python/spacewalk/satellite_tools/download.py

```diff
@@ -201,6 +201,10 @@ def __fetch_url(self, params):
                                    checksum=params['checksum']):
                 return True
 
+        retrycodes = URLGrabberOptions().retrycodes
```

Comment:
It would be a god idea to document what 14 stands for, e.g.
```suggestion
        # 14 => HTTPError (https://github.com/rpm-software-management/urlgrabber/blob/1e6d2debe79efdd1ba2f39913dc808723e51a7f7/urlgrabber/grabber.py#L757)
        retrycodes = URLGrabberOptions().retrycodes
```